client/trino-cli/src/test/java/io/trino/cli/TestFormatUtils.java

```diff
@@ -0,0 +1,353 @@
+        assertEquals(FormatUtils.formatDataRate(DataSize.ofBytes(1), Duration.valueOf("0s"), true), "0B/s");
+    }
+
+    @Test
+    public void testPluralize()
+    {
+        assertEquals(FormatUtils.pluralize("foo", 0), "foos");
+        assertEquals(FormatUtils.pluralize("foo", 1), "foo");
+        assertEquals(FormatUtils.pluralize("foo", 2), "foos");
+    }
+
+    @Test
+    public void testFormatFinalTime()
+    {
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0us")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0ns")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0ms")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0s")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0m")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0h")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("0d")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1us")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1ns")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1ms")), "0.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("10ms")), "0.01");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("100ms")), "0.10");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1s")), "1.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("10s")), "10.00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1m")), "1:00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("61s")), "1:01");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1h")), "60:00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("61m")), "61:00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("1d")), "1440:00");
+        assertEquals(FormatUtils.formatFinalTime(Duration.valueOf("25h")), "1500:00");
+    }
+
+    @Test
+    public void testFormatIndeterminateProgressBar()
+    {
+        assertEquals(FormatUtils.formatProgressBar(10, 0), "<=>       ");
+        assertEquals(FormatUtils.formatProgressBar(10, 1), " <=>      ");
+        assertEquals(FormatUtils.formatProgressBar(10, 7), "       <=>");
+        assertEquals(FormatUtils.formatProgressBar(10, 8), "      <=> ");
+        assertEquals(FormatUtils.formatProgressBar(10, 13), " <=>      ");
+        assertEquals(FormatUtils.formatProgressBar(10, 14), "<=>       ");
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "invalid count: -1")
+    public void testInvalidIndeterminateProgressBar()
+    {
+        assertEquals(FormatUtils.formatProgressBar(10, -1), "<=>       ");
```

Comment:
looks like `assertEquals` is unreachable code, since `formatProgressBar` throws, right?